types/joint.d.ts

```diff
@@ -1428,6 +1433,9 @@ export namespace dia {
         $grid: JQuery;
         $background: JQuery;
 
+        GUARDED_TAG_NAMES: string[];
+        FORM_CONTROLS_TAG_NAMES: string[];
```

Comment:
do we need to include these? we have a few other constants on Paper for which we don't have types